src/components/Tooltip/index.js

```diff
@@ -91,10 +156,15 @@ function Tooltip(props) {
         if (bounds.width === 0) {
             setIsRendered(false);
         }
-        setWrapperWidth(bounds.width);
-        setWrapperHeight(bounds.height);
-        setXOffset(bounds.x);
-        setYOffset(bounds.y);
+        const t = target.current;
+        if (!t) {
+            return;
+        }
```

Comment:
Is there a case where target is not defined?

Reply:
No, but the type system would not be able to see that, because it cannot tell that `updateBounds` will never be called before a hover event makes the tooltip visible.

Regardless, this file is not TypeScript, so I've removed it.